app/src/examples/WithClampExample.tsx

```diff
@@ -59,20 +85,20 @@ export default function AnimatedStyleUpdateExample() {
     dampingRatio: 0.075,
   };
 
-  const clampedStyle = useAnimatedStyle(() => {
+  const clampedStyleWithHoc = useAnimatedStyle(() => {
```

Comment:
What's "hoc" in this context? I know it stands for "higher-order component" but why this appears here for animated style?

Reply:
I've though of higher-order-animation and made typo. However, I'll call it "animation modifier" to match our docs